pyproject.toml

```diff
@@ -28,6 +33,8 @@ pymilvus = "^2.2.2"
 qdrant-client = {version = "^1.0.4", python = "<3.12"}
 redis = "4.5.1"
 llama-index = "0.5.4"
+azure-identity = "^1.12.0"
```

Comment:
Could you update https://github.com/openai/chatgpt-retrieval-plugin/blob/main/docs/deployment/removing-unused-dependencies.md to reflect these additions? Thank you!

Reply:
Thanks for reviewing! Updated.